sdk-lib/src/androidTest/java/cash/z/ecc/fixture/AccountFixture.kt

```diff
@@ -0,0 +1,13 @@
+package cash.z.ecc.fixture
+
+import cash.z.ecc.android.sdk.model.Account
+
+object AccountFixture {
+    val ACCOUNT_UUID = "random_uuid_16_b".toByteArray()
```

Comment:
How are these fixtures (intended to be) used? Putting aside the fact that this is probably not a valid v4 UUID, it will never correspond to a real account. If the intention of this fixture is to provide fake data in tests that never access the Rust backend, add documentation saying so.

Reply:
I slightly reworked the `AccountFixture` class, which is not now duplicated across related modules and their tests. There is only one such fixture. It now works with `UUID` instead of `String` converted to `ByteArray`. Documentation added.